indy-wrapper/src/main/java/nl/quintor/studybits/indy/wrapper/util/AsyncUtil.java

```diff
@@ -67,4 +75,8 @@
     public interface ExceptionalPredicate<T> {
         public boolean test(T t) throws Exception;
     }
+
+    public interface ExceptionalBiConsumer<T, U> {
```

Comment:
What is a `BiConsumer`? Does it have to do something with their sexuality? ;-)

Reply:
xD BiConsumers consume 2 (bi) arguments. It's standard terminology from the Java 8 API.